web/ui/static/js/targets.js

```diff
@@ -1,38 +1,62 @@
-function toggle(obj, state){
-    var icon = $(obj).find("i");
-    if (icon.length === 0 ) {
-        return;
+function toggle(button, state){
+  if (button.length === 0) { return; }
+
+  if (state) {
```

Comment:
To clarify that `state` is a boolean variable (vs. e.g. some enum) and what its `true` and `false` values mean, how about calling it `isCollapsed`?